python/model/test_plugin/test_plugin.yaml

```diff
@@ -0,0 +1,32 @@
+model:
```

Comment:
Why did you add this file instead of using `model/plugin/plugin.yaml`. I don't think there's anything 1st-/3rd-party-specific in that file so the same plugin file should work for both versions of the command.

Reply:
Gotcha
Will remove